client/my-sites/domains/domain-management/contacts-privacy/contacts-card.jsx

```diff
@@ -0,0 +1,163 @@
+import { Button, Card } from '@automattic/components';
+import { ToggleControl } from '@wordpress/components';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import { domainManagementEditContactInfo } from 'calypso/my-sites/domains/paths';
+import {
+	enableDomainPrivacy,
+	disableDomainPrivacy,
+	discloseDomainContactInfo,
+	redactDomainContactInfo,
+} from 'calypso/state/sites/domains/actions';
+import { isUpdatingDomainPrivacy } from 'calypso/state/sites/domains/selectors';
+import ContactDisplay from './contact-display';
+
+class ContactsPrivacyCard extends Component {
+	static propTypes = {
+		privateDomain: PropTypes.bool.isRequired,
+		privacyAvailable: PropTypes.bool.isRequired,
+		selectedDomainName: PropTypes.string.isRequired,
+		selectedSite: PropTypes.oneOfType( [ PropTypes.object, PropTypes.bool ] ).isRequired,
+		contactInfoDisclosureAvailable: PropTypes.bool.isRequired,
+		contactInfoDisclosed: PropTypes.bool.isRequired,
+		isPendingIcannVerification: PropTypes.bool.isRequired,
+	};
+
+	togglePrivacy = () => {
+		const { selectedSite, privateDomain, selectedDomainName: name } = this.props;
+
+		if ( privateDomain ) {
+			this.props.disableDomainPrivacy( selectedSite.ID, name );
+		} else {
+			this.props.enableDomainPrivacy( selectedSite.ID, name );
+		}
+	};
+
+	toggleContactInfo = () => {
+		const { selectedSite, contactInfoDisclosed, selectedDomainName: name } = this.props;
+
+		if ( contactInfoDisclosed ) {
+			this.props.redactDomainContactInfo( selectedSite.ID, name );
+		} else {
+			this.props.discloseDomainContactInfo( selectedSite.ID, name );
+		}
+	};
+
+	getPrivacyProtection() {
+		const { privateDomain, privacyAvailable } = this.props;
+		const { translate, isUpdatingPrivacy } = this.props;
+
+		const label = privateDomain
+			? translate( 'Privacy protection on' )
+			: translate( 'Privacy protection off' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Privacy protection: off' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-management-contact-information)